tests/acceptance/SubscriberCountShortcodeCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+use MailPoet\Test\DataFactories\Subscriber;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+class SubscriberCountShortcodeCest {
+  function createSubscriberCountPageWithShortcode(\AcceptanceTester $I) {
+    $I->wantTo('Create page with MP subscriber shortcode');
+    $segment_factory = new Segment();
+    $segment = $segment_factory->withName('SubscriberCount')->create();
+    $subscriber_factory = new Subscriber();
+    $subscriber_factory->withSegments([$segment])->create();
+    $pageTitle='OurSubscribers';
```

Comment:
Hey, I don't know why our automated tools didn't pick this up, but you should separate all binary operator `=` by a single space on both sides.

Reply:
Noted, and adjusting all the tests not yet in Master. I think the rule is being added to sniffer.